usecases/MinimumWorkingExample/dodo.py

```diff
@@ -57,3 +61,19 @@ def task_extract_processed_data_emodul():
                 'file_dep': [raw_data_file],
                 'targets': [csv_data_file],
             }
+
+#generate knowledgeGraphs
+def task_export_knowledgeGraph_emodul():
+    for f in os.scandir(metadata_emodulus_directory):
+        if f.is_file():
+            metadata_file_path = Path(f.path)
+            name_of_ttl = f.name.replace('.yaml', '.tll')
+            knowledge_graph_file = Path(knowledge_graphs_directory,
+                                        name_of_ttl)
+            yield{
+                'name': knowledge_graph_file,
+                'actions': [(generate_knowledge_graph, [metadata_file_path,
+                                                    name_of_ttl])],
```

Comment:
This generates a knowledge graph for each individual experimental test, but at the end we would like to have one single knowledgegraph for all tests used in a query. The idea was to perform a SHACL validation for each subgraph and then put them together.

Reply:
I had a misunderstanding of the task apparently. I thought that the SCHACL validation and the merge of the KGs would be done by another script. I will try to do that step but I don't really know what is the SHACL validation so I would find it helpful if someone could explain it to me or just pointed out a documentation about it.

Reply:
See my comment in #48 , we will merge this, and afterwards join the knowledge graphs. The important thing is though that all the queries work on the joint graph, not on graphs for individual single experiments.